101-data-factory-blob-2-sql/azuredeploy.json

```diff
@@ -0,0 +1,194 @@
+{
+  "contentVersion": "1.0.0.0",
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "parameters": {
+    "dataFactoryName": {
+      "type": "string",
+      "metadata": {
+        "description": "Data Factory Account name"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage Account name"
+      }
+    },
+    "storageAccountKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Storage Account Key"
+      }
+    },
+    "sourceContainer": {
```

Comment:
Prefix this and all pre-reqs below with "existing" to make it clear what is a pre-req and what is new